### PR TITLE
Catalog details panel

### PR DESCRIPTION
**Pull request by jakolehm**
https://user-images.githubusercontent.com/1446224/120588828-8a2ba800-c440-11eb-9375-d55bd51c9bbd.mov

### CODE REVIEW

File: src/renderer/components/+catalog/catalog-entity-details.scss [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+ * FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+ * COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+ * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+ .CatalogEntityDetails {
```
Comment (nevalla) [inline src/renderer/components/+catalog/catalog-entity-details.scss:22]: not sure if we already have some css variables that can be used, but we can always tweak these later.

**Review by nevalla — DISMISSED** [review]

File: src/renderer/components/+catalog/catalog-entity-details.scss (hunk 1 above)
Comment (jakolehm) [reply src/renderer/components/+catalog/catalog-entity-details.scss:22]: Good point, now it uses css variables.

**Comment by jakolehm** [comment]
@Nokel81 ptal

File: src/renderer/components/+catalog/catalog.tsx [hunk 2]
```diff
@@ -201,6 +197,7 @@ export class Catalog extends React.Component<Props> {
   renderSingleCategoryList() {
     return (
       <ItemListLayout
+        isClusterScoped
```
Comment (Nokel81) [inline src/renderer/components/+catalog/catalog.tsx]: Given that #2743 fixes this issue (and no longer has UI changes). Do we need this change?

File: src/renderer/components/+catalog/catalog.tsx [hunk 3]
```diff
@@ -287,7 +286,15 @@ export class Catalog extends React.Component<Props> {
         provideBackButtonNavigation={false}
         contentGaps={false}>
         { this.catalogEntityStore.activeCategory ? this.renderSingleCategoryList() : this.renderAllCategoriesList() }
-        <CatalogAddButton category={this.catalogEntityStore.activeCategory} />
+        { !this.selectedItem && (
+          <CatalogAddButton category={this.catalogEntityStore.activeCategory} />
+        )}
+        { this.selectedItem && (
```
Comment (Nokel81) [inline src/renderer/components/+catalog/catalog.tsx:290]: nit, maybe use ternary instead `?:`?

File: src/renderer/components/hotbar/hotbar-menu.tsx [hunk 4]
```diff
@@ -157,6 +157,7 @@ export class HotbarMenu extends React.Component<Props> {
                             className={cssNames({ isDragging: snapshot.isDragging })}
                             remove={this.removeItem}
                             add={this.addItem}
+                            size={40}
```
Comment (Nokel81) [inline src/renderer/components/hotbar/hotbar-menu.tsx:160]: Shouldn't it default to width? Or would that be too complicated?

File: src/renderer/components/+catalog/catalog.tsx (hunk 2 above)
Comment (jakolehm) [reply src/renderer/components/+catalog/catalog.tsx]: It depends on how fast we can get it merged.

File: src/renderer/components/hotbar/hotbar-menu.tsx (hunk 4 above)
Comment (jakolehm) [reply src/renderer/components/hotbar/hotbar-menu.tsx:160]: Maybe, currently it does not... this PR just made the fixed width adjustable through props. If we want this to default to container width it should be done in another PR.

Comment (Nokel81) [reply src/renderer/components/hotbar/hotbar-menu.tsx:160]: Sounds good

**Comment by jakolehm** [comment]
@nevalla @Nokel81 ptal again.

**Comment by nevalla** [comment]
> @nevalla @Nokel81 ptal again.

**Comment by jakolehm** [comment]
> > @nevalla @Nokel81 ptal again.
> 
> 

Rebase gone south 😢

**Comment by jakolehm** [comment]
@nevalla @Nokel81 ptal again again.

**Review by nevalla — DISMISSED** [review]

**Review by Nokel81 — APPROVED** [review]

**Review by nevalla — APPROVED** [review]

**Pull request merged**